content/docs/components-and-props.md

```diff
@@ -194,12 +194,12 @@ function Comment(props) {
 Next, we will extract a `UserInfo` component that renders an `Avatar` next to the user's name:
 
 ```js{3-8}
-function UserInfo(props) {
+function UserInfo({user}) {
   return (
     <div className="UserInfo">
-      <Avatar user={props.user} />
+      <Avatar user={props.author} />
```

Comment:
Since you have already removed props from params and destructured it we won't have `props`? 
PS: currently we are building beta.reactjs.org you can also use that.